python/cugraph/dask/traversal/mg_bfs_wrapper.pyx

```diff
@@ -96,14 +97,16 @@ def mg_bfs(input_df,
     if (return_distances):
         c_distance_ptr = df['distance'].__cuda_array_interface__['data'][0]
 
-    cdef bool direction = <bool> 1
-    # MG BFS path assumes directed is true
+    if depth_limit is None:
+        depth_limit = c_bfs.INT_MAX
```

Comment:
We may fix this in another PR, but I think we'll hit the 32 bit signed integer limit fairly soon for MNMG.

Reply:
updated